HolubLife/src/com/holub/life/Universe.java

```diff
@@ -103,21 +103,12 @@ public void mousePressed(MouseEvent e) {
                 );
 
 
-
+        SurroundingCells updatedSurroundingCells = new SurroundingCells();
```

Comment:
이거 관련해서 질문 드리고 싳은 사항이 있습니다.
빌더 패턴을 사용해서 SurroundingCells 객체를 생성하는 것이 목표였는데,
빌더 호출과 관련하여 어려움이 있었습니다. 정확히 말하자면 SurroundingCells Builder 생성자를 Universe.java 파일에서 찾을 수 없던 이슈가 있었습니다. 해당 이슈에 대해 아시는 분이 있다면 커멘트 부탁드리겠습니다.
AS-IS : 기본 생성자에 더미 direction 객체 담아 반환
TO-BE : 빌더 패턴 사용하여 더미 객체를 Universe.java에서 주입하여 SurroundingCells 객체 생성
static 관련하여 메서드가 보고 있는 scope가 서로 상이해서 생기는 문제인 것 같은데 나이스하게 해결되지 않는 것 같네요

Reply:
```suggestion
        SurroundingCells updatedSurroundingCells = new SurroundingCells.SurroundingCellsBuilder()
                .setNorthWest(Cell.DUMMY)
                .setNorthEast(Cell.DUMMY)
                .setNorth(Cell.DUMMY)
                .setSouthWest(Cell.DUMMY)
                .setSouthEast(Cell.DUMMY)
                .setSouth(Cell.DUMMY)
                .setWest(Cell.DUMMY)
                .setEast(Cell.DUMMY)
                .build();
```

@redjen8  위와 같이 의도 하신게 맞나요? 이 코드는 잘 수행 됩니다.